mesa/visualization/templates/js/InteractionHandler.js

```diff
@@ -0,0 +1,165 @@
+
+/**
+Mesa Visualization InteractionHandler
+====================================================================
+
+This uses the context of an additional canvas laid overtop of another canvas
+visualization and maps mouse movements to agent position, displaying any agent
+attributes included in the portrayal that are not listed in the ignoredFeatures.
+
+The following portrayal will yield tooltips with wealth, id, and pos:
```

Comment:
What happens when there is a model that doesn't have agent wealth, but has some other attributes?... let's say ... level of trust, memory, and health?